### PR TITLE
Update emuDeckRPCS3.sh

### PR DESCRIPTION
**Pull request by rawdatafeel**
* Updated migrationFlag location to "$HOME/.config/EmuDeck/.${emu}MigrationCompleted"
* Updated RPCS3_install to use installEmuAI and removed desktop creation function
    * Simplifies RPCS3 emuScript
* Updated RPCS3_emuName to lowercase
    * Matches ES-DE find rules

### CODE REVIEW

File: functions/EmuScripts/emuDeckRPCS3.sh [hunk 1]
```diff
@@ -19,26 +19,9 @@ RPCS3_install(){
 	# Migrates configurations to RPCS3 AppImage
 	RPCS3_migrate
 
-	# RPCS3 does not have a "latest" tag on their GitHub repo. Open issue said to use the below URL instead. Modified from ES-DE script
-	RPCS3_releaseMD5="$(curl -sL https://rpcs3.net/latest-appimage | md5sum | cut -d ' ' -f 1)"
-
-	local showProgress="$1"
```
Comment (SilentException) [inline functions/EmuScripts/emuDeckRPCS3.sh:25]: This should be kept, it is needed below.

**Comment by SilentException** [comment]
After #864 you can revert back emuName change and call installEmuAI with additional parameter value, it will sort the RPCS3/rpcs3 capitalization.

```
RPCS3_emuName="RPCS3"
if installEmuAI "$RPCS3_emuName" "$RPCS3_releaseURL" "rpcs3" "$showProgress"; ; then # rpcs3.AppImage - needs to be lowercase yuzu for EsDE to find it
```

**Comment by Godsbane** [comment]
merged #864 - feel free to tweak.

**Comment by SilentException** [comment]
@rawdatafeel 
In recent Dev AppImage code @dragoonDorise added migration button. But the code expects result from RPCS3_migrate function:
https://github.com/EmuDeck/emudeck-electron/commit/d7d49f4bb3360e233d6204f386e3944231c299f3#diff-19b2619ea486d5533e6b3da68d5317488553841e2eec76079b5d68afc79f915aR923
So I guess it wouldn't hurt to tweak current PR and merge it finally :)

**Comment by SilentException** [comment]
> What do I need to do? Sorry, pretty newbish still and I'm not exactly sure what that code means :)

Simply echo "true" at the end of RPCS3_migrate :)

File: functions/EmuScripts/emuDeckPCSX2QT.sh [hunk 2]
```diff
@@ -15,9 +15,9 @@ PCSX2QT_install() {
 	echo "Begin PCSX2-QT Install"
 	local showProgress="$1"
 	
-	if installEmuAI "pcsx2-Qt" "https://github.com/PCSX2/pcsx2/releases/download/v1.7.4749/pcsx2-v1.7.4749-linux-appimage-x64-Qt.AppImage" "" "$showProgress"; then #pcsx2-Qt.AppImage	
-	#if installEmuAI "pcsx2-Qt" "$(getReleaseURLGH "PCSX2/pcsx2" "Qt.AppImage")" "" "$showProgress"; then #pcsx2-Qt.AppImage
-		:
+	if installEmuAI "${PCSX2QT_emuName}" "https://github.com/PCSX2/pcsx2/releases/download/v1.7.4749/pcsx2-v1.7.4749-linux-appimage-x64-Qt.AppImage" "pcsx2-Qt" "$showProgress"; then # pcsx2-Qt.AppImage - filename capitalization matters for ES-DE to find it
+	#if installEmuAI "${PCSX2QT_emuName}" "$(getReleaseURLGH "PCSX2/pcsx2" "Qt.AppImage")" "pcsx2-Qt" "$showProgress"; then #pcsx2-Qt.AppImage
+		rm -rf $HOME/.local/share/applications/pcsx2-Qt.desktop &>> /dev/null # delete old shortcut
```
Comment (Godsbane) [inline functions/EmuScripts/emuDeckPCSX2QT.sh:20]: Why are we deleting the desktop shortcut in the installation for pcsx2qt?

Comment (Godsbane) [reply functions/EmuScripts/emuDeckPCSX2QT.sh:20]: oh - because it's the wrong capitalization?

Comment (SilentException) [reply functions/EmuScripts/emuDeckPCSX2QT.sh:20]: Yes, keeping things tidy.

**Pull request merged**